tanjun/commands.py

```diff
@@ -1079,7 +1079,7 @@ def __init__(
         )
 
         self._builder = _CommandBuilder(name, description, sort_options).set_default_permission(default_permission)
-        self._callback = injecting.CallbackDescriptor(callback)
+        self._callback = injecting.CallbackDescriptor[None](callback)
```

Comment:
mypy gets confused about what to use for `T` (`None` or `typing.Awaitable[None]`) -- I think I'll file an issue to make the error message more useful :P